xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -131,6 +131,7 @@ bool CRenderManager::Configure(const VideoPicture& picture, float fps, unsigned
 
   {
     std::unique_lock<CCriticalSection> lock(m_statelock);
+    m_isRenderInitializing = m_renderState == STATE_UNCONFIGURED;
```

Comment:
I don't see a need for this new member variable if all you do is compare `m_renderState == STATE_UNCONFIGURED`. Why not compare it directly below with `m_renderState != STATE_UNCONFIGURED`?

Reply:
will result in always true
Configure() is called only if m_renderState == STATE_CONFIGURING
instead i need to know the first init call

Reply:
I do not understand in full your statement, please notice that method
`bool CRenderManager::Configure(const VideoPicture& picture, float fps, unsigned int orientation, int buffers)`
can be called more times during playback

instead method
`bool CRenderManager::Configure()`
is called only if `m_renderState == STATE_CONFIGURING`
you cannot replace  m_isRenderInitializing  here:
https://github.com/xbmc/xbmc/blob/961d5fdeb66681ac0e7de0eea10cb36074a54d30/xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp#L225
by using `m_renderState` value state because will have a fixed value to STATE_CONFIGURING

Reply:
thanks. Looks much better to me now. 👍